api/controllers/UserController.js

```diff
@@ -71,7 +67,11 @@ module.exports = {
     userAPI
       .findUserByUsername(req.body.username)
       .then(function gotUser(user) {
-        const checkPass = passwordAPI.checkPass(req.body.password, user.encryptedPassword);
+        console.log();
+        const checkPass =
+          req.session.loggedIn !== true && req.body.password
```

Comment:
Wondering about a falsey eval of req.session.loggedIn
```suggestion
          !req.session.loggedIn && req.body.password
```

Reply:
I guess the null case is handled differently on purpose?